### DISCUSSION TITLE
cant load map to stable

### DISCUSSION THREAD

**Discussion by injox** [ppy/osu#27613] [Q&A] [answered]
I played in lazer for a while and then went back to stable, I wanted to download a new map, but the map loaded in lazer. I tried to set the score in stable to switch my player mode back to stable, but it didnt help

**Comment by Gabixel** [comment]
<sub>I'm not really understanding the issue here. Is the issue about opening the map (as title), which decided to launch lazer and not stable (which I think it's intentional behaviour, but maybe needs confirmation), or the fact that you.. "switched player mode" back to stable with no luck? (in which case I think you need to set a better / new score)</sub>

**Reply by injox** [reply]
when i open a map, that i downloaded from site, it launches lazer instead of stable

**Comment by Gabixel** [comment]
Upon re-reading, I guess you tried to open and play Stable in the hope that it would direct the opening of a beatmap file there. If that's what you meant, I don't think it works.
For now, if you want to put a beatmap on stable, you can drag it into the game window.

**Comment by jamesbt365** [comment]
The game recently changed the default so when you updated lazer it changed what the files open in by default from stable to lazer, to revert this you can do something like this:
![image](https://github.com/ppy/osu/assets/22662856/188a0095-2ab2-47aa-b962-a25142a4de80)

**Comment by peppy** [comment, marked as answer]
Running "repair folder permissions" from stable will temporarily restore stable to handling files.

If this doesn't work, uninstalling lazer should.
